doc/HOWTO_DOCUMENT.rst.txt

```diff
@@ -246,13 +246,35 @@ The sections of the docstring are:
      err_msg : str or None
          Human readable error message, or None on success.
 
-6. **Other parameters**
+6. **Yields**
+
+   Explanation of the yielded values and their types. This is relevant to
+   generators only. Similar to the **Parameters** section, except the name
+   of each return value is optional. The type of each return value is
+   always required::
+
+     Yields
+     ------
+     int
+         Description of the anonymous integer return value.
+
+   If both the name and type are specified, the **Returns** section takes the
+   same form as the **Parameters** section::
+
+     Yields
+     ------
+     err_code : int
+         Non-zero value indicates error code, or zero on success.
+     err_msg : str or None
```

Comment:
Multiple returns/yields are just tuples right?  So if we allow multiple return values, we should allow multiple yield values. It's just one tuple, in both cases.